presto-main/src/test/java/com/facebook/presto/block/AbstractTestBlock.java

```diff
@@ -371,6 +446,35 @@ protected void assertSlicePosition(Block block, int position, Slice expectedSlic
         }
     }
 
+    protected void assertSlicePositionUnchecked(Block block, int internalPosition, Slice expectedSliceValue)
+    {
+        int length = block.getSliceLengthUnchecked(internalPosition);
+        assertEquals(length, expectedSliceValue.length());
+
+        Block expectedBlock = toSingeValuedBlock(expectedSliceValue);
+        for (int offset = 0; offset < length - 3; offset++) {
+            assertEquals(block.getSliceUnchecked(internalPosition, offset, 3), expectedSliceValue.slice(offset, 3));
+            assertTrue(block.bytesEqual(internalPosition - block.getOffsetBase(), offset, expectedSliceValue, offset, 3));
+            // if your tests fail here, please change your test to not use this value
+            assertFalse(block.bytesEqual(internalPosition - block.getOffsetBase(), offset, Slices.utf8Slice("XXX"), 0, 3));
```

Comment:
nit: Why the value will always be "XXX"? -- Also don't quite understand the comment one line before? :)

Reply:
This is based off of existing, checked access.  But I agree it seems confusing, I'll change it to use a UUID which is statistically extremely unlikely to be present in a test.